includes/misc-functions.php

```diff
@@ -531,13 +531,14 @@ function edd_get_currency_name( $code = 'USD' ) {
  *
  * @since 1.0
  *
- * @param integer $n
+ * @param integer $n The number of the month.
+ * @param bool    $return_long_name Optional. Return full name of month if true. Default false.
  * @return string Short month name
  */
-function edd_month_num_to_name( $n ) {
+function edd_month_num_to_name( $n, $return_long_name = false ) {
 	$timestamp = mktime( 0, 0, 0, $n, 1, 2005 );
 
-	return date_i18n( "M", $timestamp );
+	return ! $return_long_name ? date_i18n( 'M', $timestamp ) : date_i18n( 'F', $timestamp );
```

Comment:
We can reduce this code a little by using the conditional _only_ for the first parameter value (instead of repeating the whole function twice). Example:

```suggestion
	$date_format = $return_long_name ? 'F' : 'M';
	
	return date_i18n( $date_format, $timestamp );
```

(I also flipped the `$return_long_name` logic just because I find that easier to read.)